### ISSUE TITLE
Add CSS to the bare bones Beginner Javascript Project

### ISSUE THREAD

**Issue by innatepanda** [MSCKIIT/MSCxHacktoberfest#122]
#We need more stylization to the already existing style.css
Screenshots at the time of the issue creation:
![image](https://user-images.githubusercontent.com/61987098/135715298-ebcf9c97-d2d6-4168-b8b3-db3907baab18.png)
![image](https://user-images.githubusercontent.com/61987098/135715331-39fd1a54-ab4b-43af-a5f7-eaa068f87280.png)
![image](https://user-images.githubusercontent.com/61987098/135715418-bb17c54f-3dde-499b-aeda-5cad19977921.png)

#Which tech stack does it require?
* [ ] C++ or C
* [ ] Javascript
* [ ] Python
* [ ] Node.js
* [ ] Golang
* [ ] React.js
* [ ] Ruby
* [ ] C#
* [ ] Dart
* [x] HTML and CSS

**Comment by shm-dsgn** [comment]
Please assign me to this work.

**Comment by camperjett** [comment]
can I work on this. I'm new to open source.

**Comment by CodingBlood** [comment]
> can I work on this. I'm new to open source.

Sorry shm-dsgn commented first , so issue was assigned to him. There are many more issues available you can contribute there🌝

**Comment by innatepanda** [comment]
@shm-dsgn Hi. Please create atleast one pr by tuesday midnight and don't forget to pull the recent changes. If no pr is received by tuesday midnight IST, we'll reopen this issue.

**Comment by shm-dsgn** [comment]
@innatepanda yes. Sorry for the delay. I have an exam today. Will do a PR by today only before midnight.

**Comment by innatepanda** [comment]
@shm-dsgn It's quite alright. No need to rush.

**Comment by innatepanda** [comment]
Open again.
